[Bug]: TensorFlow Zeros API at Ivy Frontend: Limitation with Paddle Backend Supports Only Dimensions 0 to 9

### Bug Explanation

When running tests for the TensorFlow Zeros API implemented at the Ivy frontend, the following error is encountered when the Paddle backend is used:
```py
OSError: Unimplemented error. Invalid dimension to be accessed. Now only supports access to dimension 0 to 9, but received dimension is 10.
```
### Suggested Fix
```py
    if ivy.current_backend_str() == "paddle":
        # Paddle only supports ndim from 0 to 9
        assume(input.shape[0] < 10)
```


### Steps to Reproduce Bug

from `CLI` run: `python3 -m pytest  ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py::test_tensorflow_zeros -B=paddle`

```py
E               OSError: Unimplemented error. Invalid dimension to be accessed. Now only supports access to dimension 0 to 9, but received dimension is 10.
E                 [/paddle/paddle/phi/core/ddim.h:55]
E               Falsifying example: test_tensorflow_zeros(
E                   frontend='tensorflow',
E                   backend_fw='paddle',
E                   on_device='cpu',
E                   input=(0, 0, 0, 0, 0, 0, 0, 0, 0, 0),
E                   dtype=['complex64'],
E                   fn_tree='ivy.functional.frontends.tensorflow.zeros',
E                   test_flags=FrontendFunctionTestFlags(
E                       num_positional_args=0,
E                       with_out=False,
E                       inplace=False,
E                       as_variable=[False],
E                       native_arrays=[False],
E                       test_compile=False,
E                       generate_frontend_arrays=False,
E                       transpile=False,
E                       precision_mode=False,
E                   ),
E               )
```

### Environment

Linux+docker+vscode

### Ivy Version

v1.1.9

### Backend

- [ ] NumPy
- [ ] TensorFlow
- [ ] PyTorch
- [ ] JAX
- [x] Paddle

### Device

CPU

--- a/ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py
+++ b/ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py
@@ -2,7 +2,6 @@
 from hypothesis import strategies as st, assume
 import numpy as np
 from tensorflow import errors as tf_errors
-import ivy
 
 # local
 import ivy_tests.test_ivy.helpers as helpers
@@ -2382,9 +2381,9 @@ def test_tensorflow_while_loop(
     input=helpers.get_shape(
         allow_none=False,
         min_num_dims=0,
-        max_num_dims=10,
+        max_num_dims=9,
         min_dim_size=0,
-        max_dim_size=10,
+        max_dim_size=9,
     ),
     dtype=helpers.get_dtypes("valid", full=False),
 )
@@ -2398,9 +2397,6 @@ def test_tensorflow_zeros(
     fn_tree,
     on_device,
 ):
-    if ivy.current_backend_str() == "paddle":
-        # Paddle only supports ndim from 0 to 9
-        assume(input.shape[0] < 10)
     helpers.test_frontend_function(
         shape=input,
         input_dtypes=dtype,

--- a/ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py
+++ b/ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py
@@ -2,6 +2,7 @@
 from hypothesis import strategies as st, assume
 import numpy as np
 from tensorflow import errors as tf_errors
+import ivy
 
 # local
 import ivy_tests.test_ivy.helpers as helpers
@@ -2397,6 +2398,9 @@ def test_tensorflow_zeros(
     fn_tree,
     on_device,
 ):
+    if ivy.current_backend_str() == "paddle":
+        # Paddle only supports ndim from 0 to 9
+        assume(input.shape[0] < 10)
     helpers.test_frontend_function(
         shape=input,
         input_dtypes=dtype,
@@ -2412,7 +2416,7 @@ def test_tensorflow_zeros(
 @handle_frontend_test(
     fn_tree="tensorflow.zeros_like",
     dtype_and_x=helpers.dtype_and_values(
-        available_dtypes=helpers.get_dtypes("numeric")
+        available_dtypes=helpers.get_dtypes("numeric"),
     ),
     dtype=helpers.get_dtypes("numeric", full=False),
     test_with_out=st.just(False),

